packages/fast-usdc-deploy/test/chain-impact.test.ts

```diff
@@ -169,17 +175,6 @@ test.serial('access relevant kernel stats after bootstrap', async t => {
   observations.push({ id: 'post-boot', ...relevant });
 });
 
-test.serial('prep for contract deployment', async t => {
-  const { sim } = t.context;
-  await t.notThrowsAsync(sim.beforeDeploy(t));
```

Comment:
why take out the `beforeDeploy` step?

Reply:
The oracle provision it was doing was rolled up into `beforeIterations`